testing/mu/APKBUILD

```diff
@@ -1,40 +1,41 @@
 # Contributor: Sören Tempel <soeren+alpine@soeren-tempel.net>
 # Contributor: stef <l0ls0fo2i@ctrlc.hu>
+# Contributor: Eivind Uggedal <eu@eju.no>
 # Maintainer:
 pkgname=mu
-pkgver=1.0
+pkgver=1.2
+_pkgver=1.2.0
 pkgrel=0
 pkgdesc="A collection of utilities for indexing and searching Maildirs"
 arch="x86 x86_64 ppc64le"
 url="http://www.djcbsoftware.nl/code/mu"
 license="GPL-3.0"
```

Comment:
Please clarify the license and use the SPDX identifier - https://spdx.org/licenses/